corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -188,64 +188,64 @@ def consumption_entry(case_id, product_id, section_id):
 
 
 def get_case_payload(domain, stock_settings, version, user, last_sync, synclog):
-        response = StringRestoreResponse()
```

Comment:
Yeah, check it with white space turned off: https://github.com/dimagi/commcare-hq/pull/5934/files?w=0